common/expression/tests/it/main.rs

```diff
@@ -362,6 +519,30 @@ fn builtin_functions() -> FunctionRegistry {
         |lhs, rhs| lhs + rhs,
     );
 
+    registry.register_2_arg::<NumberType<i32>, NumberType<i32>, NumberType<i32>, _, _>(
+        "minus",
+        FunctionProperty::default(),
+        |lhs, rhs| {
+            Some(IntDomain {
+                min: lhs.min.checked_add(rhs.min).unwrap_or(i32::MAX as i64),
+                max: lhs.max.checked_add(rhs.max).unwrap_or(i32::MAX as i64),
+            })
+        },
+        |lhs, rhs| lhs - rhs,
+    );
+
+    registry.register_2_arg::<NumberType<i64>, NumberType<i64>, NumberType<i64>, _, _>(
+        "multiply",
+        FunctionProperty::default(),
+        |lhs, rhs| {
+            Some(IntDomain {
+                min: lhs.min.checked_add(rhs.min).unwrap_or(i64::MAX),
+                max: lhs.max.checked_add(rhs.max).unwrap_or(i64::MAX),
+            })
```

Comment:
This domain calculation for `multiply` is even more complicated. You may return `None` (which will extend the domain to the full range of `i64`) if you want to leave it for later.